### PR TITLE
Hmka/migrations nullables

### PR DESCRIPTION
**Pull request by maroshmka**
### Changelog
- remove `use_prefix` for result_table. User can't change name of the table
- remove Enum for Checks. Prefix of the table is in `_table_prefix` attribute on models
- change migration template. do migration for all the models separately if needed
- refactor migration tests. introduce MigrationTestCase class.

*Migration needed*
- add new migration. Add nullables + default time_filter ('not_set') for QualityCheck and
  ConsistencyCheck models.

### Recommended reading

1.  contessa/alembic/README
2. contessa/alembic/script.py.mako
3. contessa/alembic/versions/a179e5ca0ad2_set_defaults_and_nullables.py
4. contessa/alembic/versions/480e6618700d_change_type_to_name.py
5. contessa/models.py
6. runners
7. test/integration/test_migration_runner.py:: MigrationTestCase
8. other tests

### CODE REVIEW

File: contessa/alembic/script.py.mako [hunk 1]
```diff
@@ -15,10 +17,60 @@ down_revision = ${repr(down_revision)}
...
+
+    print("Migration Quality Check")
+    for table in get_quality_tables(QualityCheck._table_prefix):
+        print(f"Migrate table {table}")
+
+    print("Migration Consistency Check")
+    for table in get_quality_tables(ConsistencyCheck._table_prefix):
+        print(f"Migrate table {table}")
+
     ${upgrades if upgrades else "pass"}
```
[the remark this answers is not in the document]
Comment (maroshmka) [reply contessa/alembic/script.py.mako]: couldn't find it either, so I rather kept it there 😄

File: docs/changelog.rst [hunk 2]
```diff
@@ -1,6 +1,46 @@
...
+
+*Migration needed*
+- add new migration. Add nullables + default time_filter ('not_set') for QualityCheck and
+  ConsistencyCheck models.
+
+Steps:
+
+1. Update default values for time_filter
+```sql
+update {t}
```
[the remark this answers is not in the document]
Comment (maroshmka) [reply docs/changelog.rst]: thats actually really good point :D gonna add it there

File: contessa/alembic/script.py.mako (hunk 1 above)
[the remark this answers is not in the document]
Comment (junckritter) [reply contessa/alembic/script.py.mako]: Ok, fine for me then :)

**Comment by junckritter** [comment]
Lookgs good

**Pull request merged**